llvm/unittests/DebugInfo/BTF/CMakeLists.txt

```diff
@@ -9,5 +9,3 @@ add_llvm_unittest(DebugInfoBTFTests
   )
 
 target_link_libraries(DebugInfoBTFTests PRIVATE LLVMTestingSupport)
-
-set_property(TARGET DebugInfoBTFTests PROPERTY FOLDER "Tests/UnitTests/DebugInfoTests")
```

Comment:
The folder is now set in `add_llvm_unittest`. it is not necessary anymore to assign it in the CMakeLists.txt. This means that it will not be put into a "DebugInfoTests" folder because `add_llvm_unittest` doesn't know about it. However, these subfolders were hard to keep consistent (only users of cmake IDE project generators would even see them and likely ignore them when they make a change). I only kept them when there was another mechanism such as `add_llvm_debuginfo_unittest` function that would enforce it.